.github/frontend/03-ui-ux.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Frontend - Phase 03
+title: Mohaymen[YEAR]-FE-Phase03-Team[TEAM_NUMBER]
+labels: Frontend
+about: This issue template is based on frontend phase 03 pilot
+---
+
+-   [ ] Introduction
+-   [ ] Learn
+  -   [ ] Difference UI/UX
+  -   [ ] Basic Principles
```

Comment:
- [ ] Learn Basic Principles of UI/UX
- [ ] understand the difference of UI and UX